integration/db/src/main/resources/changelogs/changelog-2024-01-25-11-31-18.yml

```diff
@@ -0,0 +1,12 @@
+databaseChangeLog:
+- changeSet:
+    id: 1706157085386-1
+    author: HTG (generated)
+    changes:
+    - addColumn:
```

Comment:
Another benefit of changelog per version is that this change can be included in the "createTable" command in the previous file instead of being "fragmented" like this.